### PR TITLE
pcmdi_metrics_driver.py dealing with ocean vertical levels

### PR DESCRIPTION
**Pull request by jservonnat**
Version of the pcmdi_metrics_driver.py that deals with ocean vertical levels (same mechanism as for the atmosphere); requires to and so pre-computed on standard vertical levels.
I've made a separate driver since I haven't tested the new version yet (demo failing).

### CODE REVIEW

File: src/python/pcmdi/scripts/pcmdi_metrics_driver_OceanLevels.py [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+
+# Statistical tracker
+cdat_info.pingPCMDIdb("pcmdi_metrics", "pcmdi_metrics_driver")
+
+# Before we do anything else we need to create some units
+# Salinity Units
+unidata.udunits_wrap.init()
+
+# Create a dimensionless units named dimless
+unidata.addDimensionlessUnit("dimless")
```
Comment (durack1) [inline src/python/pcmdi/scripts/pcmdi_metrics_driver_OceanLevels.py]: Nice, we were going about this another way.. But this looks promising @dnadeau4 @doutriaux1

**Comment by durack1** [comment]
@jservonnat thanks for submitting this.. We'll need to get the test passing before considering merging (not sure what the issue is). We'll also have to figure out how best to integrate your contribution, I'm not sure that another (separate) driver file is the best path forward.. But we'll have to think about this as it's almost 1000 lines of code..

**Comment by durack1** [comment]
@jservonnat the error could just be because you haven't `flake8-ed` your file, which is a trivial one line command (it cleans up the formatting) @doutriaux1 is the implementer of this, and he can defend his stance:
```
======================================================================
FAIL: flake8 (test_flake8.TestFlake8)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/PCMDI/pcmdi_metrics/test/test_flake8.py", line 36, in flake8
    self.assertTrue(out == "")
AssertionError: False is not true
----------------------------------------------------------------------
Ran 13 tests in 277.939s
FAILED (failures=1)
```

**Comment by doutriaux1** [comment]
@gleckler1 @jservonnat is this still something we want?

**Comment by doutriaux1** [comment]
@jservonnat @gleckler1 what do we do about this one?

**Comment by gleckler1** [comment]
@doutriaux1  We can close for now.  When we get around to this we will do it differently.

**Pull request closed without merging**